package.json

```diff
@@ -23,54 +23,54 @@
     "0ungit-credentials-helper": "./bin/credentials-helper"
   },
   "dependencies": {
-    "@primer/octicons": "^9.2.0",
-    "bluebird": "~3.5.4",
-    "blueimp-md5": "~2.10.0",
+    "@primer/octicons": "~9.4.0",
+    "bluebird": "~3.7.2",
+    "blueimp-md5": "~2.12.0",
     "body-parser": "~1.19.0",
-    "color": "~3.1.1",
+    "color": "~3.1.2",
     "cookie-parser": "~1.4.4",
     "crossroads": "~0.12.2",
     "dedent": "^0.7.0",
     "diff2html": "~3.1.2",
     "dnd-page-scroll": "0.0.4",
-    "express": "~4.16.4",
-    "express-session": "~1.16.1",
-    "getmac": "~1.4.6",
+    "express": "~4.17.1",
+    "express-session": "~1.17.0",
+    "getmac": "~5.1.0",
     "hasher": "~1.2.0",
-    "ignore": "~5.1.1",
+    "ignore": "~5.1.4",
     "jquery": "^3.4.1",
     "jquery-ui-bundle": "~1.12.1",
     "just-detect-adblock": "~1.0.0",
-    "knockout": "~3.5.0",
+    "knockout": "~3.5.1",
     "latest-version": "~5.1.0",
     "locks": "~0.2.2",
-    "lodash": "~4.17.11",
-    "memorystore": "~1.6.1",
-    "mkdirp": "~0.5.1",
+    "lodash": "~4.17.15",
+    "memorystore": "~1.6.2",
+    "mkdirp": "~1.0.3",
     "moment": "~2.24.0",
-    "node-cache": "~4.2.0",
+    "node-cache": "~5.1.0",
     "nprogress": "^0.2.0",
-    "open": "~6.1.0",
-    "passport": "~0.4.0",
+    "open": "~7.0.2",
+    "passport": "~0.4.1",
     "passport-local": "~1.0.0",
-    "raven-js": "~3.27.0",
+    "raven-js": "~3.27.2",
     "rc": "~1.2.8",
-    "rimraf": "~2.6.3",
-    "semver": "~6.0.0",
-    "serve-static": "~1.13.2",
+    "rimraf": "~3.0.2",
+    "semver": "~7.1.3",
+    "serve-static": "~1.14.1",
     "signals": "~1.0.0",
     "snapsvg": "~0.5.1",
-    "socket.io": "~2.2.0",
-    "superagent": "~5.0.5",
-    "temp": "~0.9.0",
-    "winston": "2.4.0",
-    "yargs": "~13.2.2"
+    "socket.io": "~2.3.0",
+    "superagent": "~5.2.2",
+    "temp": "~0.9.1",
+    "winston": "3.2.1",
```

Comment:
Why is this the only pinned dependency?

Reply:
Thanks for catching that. It shouldn't be pinned anymore because I adjusted the code to the new API.